.github/workflows/e2e-ci.yaml

```diff
@@ -149,203 +149,198 @@ jobs:
           chmod +x redli
           ./redli --tls -h $REDIS_HOSTNAME.redis.cache.windows.net -p 6380 -a $REDIS_PASSWORD FLUSHALL
 
-  # Build and deploy Facade Azure Function
-  # Ideally, use OIDC authentication to deploy the function
-  # but this is not yet supported.
-  # See: https://github.com/Azure/functions-action/issues/153
   deploy_facade_function:
     environment:
       name: CI
     needs:
-    - env_var
+      - env_var
     name: Build and Deploy Facade Azure Function
     runs-on: ubuntu-latest
     if: needs.env_var.outputs.RunE2ETestsOnly != 'true' && needs.env_var.outputs.StopFullCi != 'true'
     env:
-      AZURE_FUNCTIONAPP_PACKAGE_PATH: 'LoRaEngine/LoraKeysManagerFacade/'
+      AZURE_FUNCTIONAPP_PACKAGE_PATH: "LoRaEngine/LoraKeysManagerFacade/"
 
     steps:
-    - name: Az CLI login
-      uses: azure/login@v1
-      with:
-        client-id: ${{ secrets.AZURE_CLIENT_ID }}
-        tenant-id: ${{ secrets.AZURE_TENANT_ID }}
-        subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
-
-    - uses: actions/checkout@v3
-      name: Checkout current branch
-
-    - uses: actions/setup-dotnet@v3.0.3
-      with:
-        dotnet-version: '6.0.x'
-
-    - name: 'Resolve Project Dependencies Using Dotnet'
-      shell: bash
-      run: |
-        pushd './${{ env.AZURE_FUNCTIONAPP_PACKAGE_PATH }}'
-        dotnet build --configuration ${{ env.BUILD_CONFIGURATION }} --output ./output
-        popd
-
-    - name: Read Secrets from keyvault
-      uses: ./.github/actions/read_keyvault
-      with:
-        KEY_ARRAY: ('FUNCTION-FACADE-APP-NAME', 'AZURE-FUNCTIONAPP-PUBLISH-PROFILE', 'AZURE-RG')
```

Comment:
Did we remove this secret from keyvault?

Reply:
Not yet, as the PR is not merged, I did not want to block the E2E pipeline, I will do it when the PR will be merged

Reply:
Here is the Link to the PR Run (sorry should have included from the get go). https://github.com/Azure/iotedge-lorawan-starterkit/actions/runs/3417834128

The status shows red, but it seems to be because of a failing condition as it tries to update a non existing PR...
I am running the CI on this PR

Reply:
Created #1983 to detail issue above

Reply:
The secret was removed from the pipeline